app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files_names = command.split()
```

Comment:
not good name of this split (`cp` - is it file name too ?)

Reply:
Hi! How about "arguments"? Is this good name?